### PR TITLE
Add a loader for Shanbay word

### PR DESCRIPTION
**Pull request by xingty**
仓库默认的扇贝读取的是打卡数据，我新增了一个扇贝单词的loader，可以读取每天学习单词的数量。这条API的优点跟Calendar的API一样，不需要登录即可读取数据。

效果:

### CODE REVIEW

**Comment by yihong0618** [comment]
nice

**Comment by yihong0618** [comment]
@xingty I think we can drop shanbay loader and just keep your loader since if one try to remember words they must have checked in.

**Comment by xingty** [comment]
> @xingty I think we can drop shanbay loader and just keep your loader since if one try to remember words they must have checked in.

Alright! I will make some modifications regarding this and push the updated commits a bit later

**Comment by yihong0618** [comment]
> > @xingty I think we can drop shanbay loader and just keep your loader since if one try to remember words they must have checked in.
> 
> Alright! I will make some modifications regarding this and push the updated commits a bit later

and rename your loader to shanbay loader

**Comment by xingty** [comment]
> and rename your loader to shanbay loader

👌

**Comment by xingty** [comment]
Hi, @yihong0618. I have updated the PR with a new commit.

File: github_poster/loader/__init__.py [hunk 1]
```diff
@@ -96,5 +96,5 @@
     "CovidLoader",
     "TodoistLoader",
     "OpenLanguageLoader",
-    "ChatGPTLoader",
+    "ChatGPTLoader"
```
Comment (yihong0618) [inline github_poster/loader/__init__.py:99]: why drop the `,`

File: github_poster/loader/shanbay_loader.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-from github_poster.loader.config import SHANBAY_CALENDAR_API
+from github_poster.loader.config import SHANBAY_WORD_API
 
 
 class ShanBayLoader(BaseLoader):
-    track_color = "#33C6A4"
-    unit = "days"
+    track_color = "#ADD8E6"
+    # special1 = '#009688'
+    # special2 = '#007BFF'
```
Comment (yihong0618) [inline github_poster/loader/shanbay_loader.py:11]: drop the comments maybe better?

File: github_poster/loader/shanbay_loader.py [hunk 3]
```diff
@@ -23,29 +26,54 @@ def add_loader_arguments(cls, parser, optional):
...
+            ipp = data["ipp"] or 20
+            if len(objects) < ipp:
+                break
+            date = objects[-1]["date"]
+            year = int(date[0:4])
+            if year < self.from_year:
+                break
+
+            page += 1
+            time.sleep(0.05)
```
Comment (yihong0618) [inline github_poster/loader/shanbay_loader.py:57]: make it 0.5 let us follow the spider rule

**Comment by yihong0618** [comment]
@xingty please fix the lint

pip install -U black
black .

**Comment by yihong0618** [comment]
@xingty  Thank you very much, shanbay loader is much better now.

**Pull request merged**